aztec/src/main/kotlin/org/wordpress/aztec/AztecTextFormat.kt

```diff
@@ -37,5 +37,28 @@ enum class AztecTextFormat : ITextFormat {
     FORMAT_MONOSPACE,
     FORMAT_CODE,
     FORMAT_MARK,
-    FORMAT_HIGHLIGHT
+    FORMAT_HIGHLIGHT;
+
+    companion object {
+        /**
+         * Marks all the inline text formats which could be applied at once.
+         */
+        val inclusiveInlineStyles: List<AztecTextFormat> = listOf(
+                FORMAT_BOLD,
+                FORMAT_STRONG,
+                FORMAT_ITALIC,
+                FORMAT_EMPHASIS,
+                FORMAT_CITE,
+                FORMAT_STRIKETHROUGH,
+                FORMAT_UNDERLINE,
+                FORMAT_MARK
+        )
+
+        /**
+         * Marks all the inline text formats which are exclusive. When applied they cancel the other text formats.
+         */
+        val exclusiveInlineStyles: List<AztecTextFormat> = listOf(
+                FORMAT_CODE
+        )
```

Comment:
I agree this is not perfect but I'm not sure this could be done in a better way. We can't tie `ToolbarAction` to `TextFormat` because. Toolbar action is only related to the toolbar and doesn't cover all the text formats. For example `Mark` cannot be added from the toolbar but it's still supported by the editor. On the other hand there are other text formats which are not inline. 
In the end I've changed the behaviour so that we only need to define the `exclusive` formats and the logic is that inclusive formats are inheriting from IAztecInlineStyle and are not `exclusive`. I think this solution is better.

Reply:
good suggestion! I've used the `IAztecExclusiveInlineSpan` interface and I like this solution much better than before!